### PR TITLE
Add restore button path, some refactoring

### PR DESCRIPTION
**Pull request by MichelMichels**
Hi, I don't know if you'd accept PR's but I've changed the restore button to the actual restore icon. I also update Jot and my XAML styler auto formatted some XAML.

If you don't accept PR's, no hard feelings!

### CODE REVIEW

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml [hunk 1]
```diff
@@ -50,69 +47,76 @@
...
-                        RenderOptions.EdgeMode="Aliased"
-                        Style="{StaticResource TitlebarMinimizeMaximizeButtonStyle}">
+                <!--  Title  -->
+                <TextBlock
+                    Grid.Column="0"
+                    Margin="8,0,0,0"
+                    VerticalAlignment="Center"
+                    Text="{Binding Title, RelativeSource={RelativeSource AncestorType=Window}}" />
+
+
```
Comment (sygmond) [inline src/WindowChromeMultiscreen.UI/MainWindow.xaml:68]: Perfection my friend :)) Two lines at the price of one :)) Kidding, not important.

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml [hunk 2]
```diff
@@ -1,46 +1,43 @@
...
+    AllowsTransparency="True"
+    AutomationProperties.IsOffscreenBehavior="Onscreen"
+    RenderOptions.BitmapScalingMode="Fant"
+    SizeToContent="Manual"
+    SnapsToDevicePixels="True"
+    StateChanged="Window_StateChanged"
+    Style="{StaticResource ChromeTitlebarStyle}"
+    UseLayoutRounding="True"
+    WindowStyle="None"
+    mc:Ignorable="d">
```
Comment (sygmond) [inline src/WindowChromeMultiscreen.UI/MainWindow.xaml:25]: I have to ask. What tool you use for XAML auto format? I'm trying to see the logic in the new format above. Looks like some lines are in alphabetic order (line 5 to 8, 16 to 24) and some other not. Obviously Width, Height, MinWidth and MinHeight order is better now and if these would be in alphabetic order would not look good. That's why I'm asking, is the tool automatic or semi-automatic? Thanks.

File: WindowChromeMultiscreen.sln [hunk 3]
```diff
@@ -1,11 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WindowChromeMultiscreen.UI", "src\WindowChromeMultiscreen.UI\WindowChromeMultiscreen.UI.csproj", "{C153CBDC-4093-4905-8B52-F81BF03834D1}"
+# Visual Studio Version 17
+VisualStudioVersion = 17.7.34221.43
```
Comment (sygmond) [inline WindowChromeMultiscreen.sln:4]: Thanks for the correction. I'm using Rider and probably for Visual Studio is different. I switch to Rider in the last year and I love it. Didn't thought I would say that ever but is amazing for Blazor/Web, not for WPF.

File: src/WindowChromeMultiscreen.UI/Styles/Icons.xaml [hunk 4]
```diff
@@ -1,12 +1,7 @@
...
+    <PathGeometry x:Key="Restore" Figures="M2.1,0v2H0v8.1h8.2v-2h2V0H2.1z M7.2,9.2H1.1V3h6.1V9.2z M9.2,7.1h-1V2H3.1V1h6.1V7.1z" />
```
Comment (sygmond) [inline src/WindowChromeMultiscreen.UI/Styles/Icons.xaml:5]: Haha, I was looking for this one and at the time of making this project to help someone on github... I was such in a hurry that I coudn't find the right path. Thanks.

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml [hunk 5]
```diff
@@ -50,69 +47,76 @@
...
                     </Style>
                 </Grid.Style>
 
-                <!--Minimize Button-->
-                <Button Grid.Column="2"
-                        Click="OnMinimizeButtonClick"
-                        RenderOptions.EdgeMode="Aliased"
-                        Style="{StaticResource TitlebarMinimizeMaximizeButtonStyle}">
+                <!--  Title  -->
+                <TextBlock
```
Comment (sygmond) [inline src/WindowChromeMultiscreen.UI/MainWindow.xaml:62]: I don't know how I missed the Title, nice.

**Review by sygmond — COMMENTED** [review]
Hi, thank you very much for the contribution! Nice changes and are welcomed. In fact I'm pretty happy that someone found this little project useful (hopefully). :D

I left you some comments, if you have time, take a look.

File: WindowChromeMultiscreen.sln (hunk 3 above)
Comment (MichelMichels) [reply WindowChromeMultiscreen.sln:4]: This was done automatically by Visual Studio, I can't take credit for it 😄

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml (hunk 2 above)
Comment (MichelMichels) [reply src/WindowChromeMultiscreen.UI/MainWindow.xaml:25]: I'm using [XamlStyler](https://github.com/Xavalon/XamlStyler) with the default settings. The formatting is done automatically on save.

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml (hunk 1 above)
Comment (MichelMichels) [reply src/WindowChromeMultiscreen.UI/MainWindow.xaml:68]: > Perfection my friend :)) Two lines at the price of one :)) Kidding, not important.

I think you have a good point here. I must've missed it.

File: src/WindowChromeMultiscreen.UI/MainWindow.xaml (hunk 2 above)
Comment (MichelMichels) [reply src/WindowChromeMultiscreen.UI/MainWindow.xaml:25]: The documentation explains the reordering [here](https://github.com/Xavalon/XamlStyler/wiki/Attribute-Reordering#attribute-ordering-rule-groups).

**Pull request merged**